sandbox_template.yaml

```diff
@@ -375,3 +375,34 @@ Resources:
           - ShouldUseMutualTLS
           - true
           - !Ref "AWS::NoValue"
+      AccessLogSettings:
+        DestinationArn: !GetAtt ApiGwAccessLogs.Arn
+        Format: "{ \
+          'requestTime': '$context.requestTime', \
```

Comment:
do we need quotes around the key and value? Currently the logs seem to be going into cloudwatch as a string that just happens to contain json, rather than as a json object

Reply:
Updated to " with appropriate escaping. Logs now properly coming through as json (see attached image).
![api_logs](https://github.com/NHSDigital/prescriptionsforpatients/assets/13315158/c9174333-c8f3-4585-8f75-12293ea5ef5c)